aiodownloader/downloader.py

```diff
@@ -55,7 +59,7 @@ async def get_size(self) -> int:
                     self.size = int(resp.headers['Content-Length'])
 
                 else:
-                    raise aiohttp.errors.HttpProcessingError(
+                    raise Exception(
```

Comment:
This library is not maintained, but I can merge this.

For why I get, this exceptions are deprecated, however they should have replaced with something equivalent. Raising `Exception` is not recommended because it is a very broad exception. Could you replace this exceptions with something more specific?